components/listingsPage/filters.js

```diff
@@ -255,11 +263,19 @@ const Filters = ({ t }) => {
                 />
               )
             )}
-            <Heading level={4}>
+            <Heading level={4} color="#217D17">
```

Comment:
I still think that this heading should be a `legend` of a different `FormField` with the single of annual radio buttons

Reply:
It is a sub of InsuranceType and i think its the best way to vary their sizes

Reply:
I agree with @andreferreiradlw on this: semantically-speaking I believe those should be two `fieldsets`. The styling issue should be considered separately to that.

Reply:
Done Sir